backend/src/main/java/com/bakdata/conquery/models/concepts/filters/specific/AbstractSelectFilter.java

```diff
@@ -85,20 +83,31 @@ public void addImport(Import imp) {
 		if (values == null) {
 			values = new HashSet<>();
 		}
-		values.addAll(Sets.newHashSet(((AStringType) getColumn().getTypeFor(imp)).iterator()));
+
+		for (String value : ((AStringType<?>) getColumn().getTypeFor(imp))) {
```

Comment:
Das `getTypeFor(...)` liest sich komisch. Hätte jetzt nicht erwartet etwas zum Iterieren zurückzubekommen.

Reply:
AStringType ist auch ein Iterable String for the Lulz